pkg/apis/eventing/v1alpha1/rabbitmq_validation_test.go

```diff
@@ -78,17 +78,6 @@ func TestValidateRabbitmqBrokerConfig(t *testing.T) {
 			wantErr:  true,
 			isUpdate: false,
 		},
-		"missing rabbitmqClusterReference.namespace": {
```

Comment:
we can revert this change as well since the source code changes have been reverted for this file it looks like.